src/Accounts/Accounts/AzureRmAlias/Mappings.json

```diff
@@ -947,14 +947,15 @@
     "New-AzFrontDoorFrontendEndpointObject": "New-AzureRmFrontDoorFrontendEndpointObject",
     "New-AzFrontDoorHealthProbeSettingObject": "New-AzureRmFrontDoorHealthProbeSettingObject",
     "New-AzFrontDoorLoadBalancingSettingObject": "New-AzureRmFrontDoorLoadBalancingSettingObject",
-    "New-AzFrontDoorMatchConditionObject": "New-AzureRmFrontDoorMatchConditionObject",
-    "New-AzFrontDoorCustomRuleObject": "New-AzureRmFrontDoorCustomRuleObject",
-    "New-AzFrontDoorManagedRuleObject": "New-AzureRmFrontDoorManagedRuleObject",
-    "New-AzFrontDoorFireWallPolicy": "New-AzureRmFrontDoorFireWallPolicy",
-    "Get-AzFrontDoorFireWallPolicy": "Get-AzureRmFrontDoorFireWallPolicy",
-    "Set-AzFrontDoorFireWallPolicy": "Set-AzureRmFrontDoorFireWallPolicy",
-    "Remove-AzFrontDoorFireWallPolicy": "Remove-AzureRmFrontDoorFireWallPolicy",
-    "New-AzFrontDoorRuleGroupOverrideObject": "New-AzureRmFrontDoorRuleGroupOverrideObject",
+    "New-AzFrontDoorWafMatchConditionObject": "New-AzureRmFrontDoorWafMatchConditionObject",
```

Comment:
You should change only the Az name of the cmdlet here, and not the AzureRm cmdlet name